DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -162,33 +159,58 @@ public void PromoteDelegateToAdmin(AdminRoles adminRoles, int categoryId, int de
 
             var adminUser = userDataService.GetAdminUserByEmailAddress(delegateUser.EmailAddress);
 
-            if (adminUser != null)
+            if (adminUser?.Active is false && adminUser.CentreId == delegateUser.CentreId)

```

Comment:
What is the difference between `is false` and `== false`?

Reply:
In this case, basically nothing, so I've changed it to `==` for consistency as that seems to what be what we use elsewhere.